File: pset1/cash/cash.c
#include <cs50.h>
#include <stdio.h>
#include <math.h>

int main(void)
{
    //Great variable names!
    // No comments at all :`(
    int coins = 0;
    float number;
    int change;
    do
    {
        number = get_float("Change owed: ");
        change = round(number * 100);
    }
    while (change <= 0);

    while (change >= 25)
    {
        // Good use of shorthand here!
        coins++;
        //try using the shorthand here too, change -= 25;
        change = change - 25;
    }
    while (change >= 10)
    {
        coins++;
        change = change - 10;
    }
    while (change >= 5)
    {
        coins++;
        change = change - 5;
    }
    while (change >= 1)
    {
        coins++;
        change = change - 1;
    }
    printf("%i\n", coins);
}

File: pset6/sentimental/cash.py
from cs50 import get_float
coins = 0
while True:
    number = get_float("Change owed: ")
    break
change = round(number * 100)
while change >= 25:
    coins += 1
    change -= 25
while change >= 10:
    coins += 1
    change -= 10
while change >= 5:
    coins += 1
    change -= 5
while change >= 1:
    coins += 1
    change -= 1
print(coins)






